cogs/meta.py

```diff
@@ -58,7 +58,7 @@ async def reload(self, ctx, *modules):
             await ctx.message.delete()
 
         if not modules:
-            modules = [f[:-3] for f in os.listdir("cogs") if isfile(join("cogs", f))]
+            modules = [f[:-3] for f in os.listdir("cogs") if isfile(join("cogs", f) and f != "__init__.py")]
```

Comment:
instead of having this workaround why not just put __init__.py somewhere else?

or does it have to be in the cogs file